Source/Charts/Charts/PieChartView.swift

```diff
@@ -72,6 +72,8 @@ open class PieChartView: PieRadarChartViewBase
     /// maximum angle for this pie
     private var _maxAngle: CGFloat = 360.0
 
+    private var _drawSliceTextMinimumAngle: CGFloat = 0.0
```

Comment:
This should be one `open` variable with a `didSet`. The comment should indicate whether this is degrees or radians. The name is weird; it sounds like an angle value is going to be drawn. Perhaps something like `minimumAngleToDrawSliceText` or `sliceTextDrawingThreshold`. I'm open to suggestions.